paper/paper.md

```diff
@@ -16,48 +16,201 @@ authors:
+
+
+# A Harmonized and Integrated Workflow
+
+A key design principle of the **easystats** ecosystem is the harmonization and integration of different packages into a simple, sequential workflow. The typical workflow for a statistical analysis using `{easystats}` starts with importing data and bringing the data into shape for the next step—fitting a model—and then sequentially using different functions to obtain a comprehensive understanding of the model. This can include checking the model's parameters, performance metrics, specific effect sizes [e.g., @ben2023phi], and statistical outliers [@theriault2024check], as well as obtaining publication-ready figures and written summaries of the results.
+
+Let's demonstrate this with an example, where the user starts by preparing some data and then fits a logistic mixed effects model:
+
+
+``` r
+# we don't load each package individually,
+# but rather the entire ecosystem
+library(easystats)
+data(coffee_data, package = "modelbased")
+
+# dichotomize outcome variable
+coffee_data$alertness <- categorize(coffee_data$alertness, lowest = 0)
+
+# rename variable
+coffee_data <- data_rename(coffee_data, select = c(treatment = "coffee"))
+
+# fit mixed model
+model <- glmmTMB::glmmTMB(
+  alertness ~ treatment + (1 | ID),
+  data = coffee_data,
+  family = binomial()
+)
+```
+
+The `model` object can then be passed to functions from different **easystats** packages. For instance, the user can get a summary of the model parameters using the `{parameters}` package:
+
+
+``` r
+model_parameters(model)
+#> # Fixed Effects
+#> 
+#> Parameter           | Log-Odds |   SE |        95% CI |     z |     p
+#> ---------------------------------------------------------------------
+#> (Intercept)         |     0.13 | 0.26 | [-0.37, 0.64] |  0.52 | 0.606
+#> treatment [control] |    -0.27 | 0.37 | [-0.99, 0.45] | -0.73 | 0.466
+#> 
+#> # Random Effects
+#> 
+#> Parameter          | Coefficient |           95% CI
+#> ---------------------------------------------------
+#> SD (Intercept: ID) |        0.05 | [0.00, 9.86e+28]
+```
+
+Then, the performance of the model can be assessed with the `{performance}` package:
+
+
+``` r
+model_performance(
+  model,
+  metrics = c("AIC", "BIC", "R2")
```

Comment:
Why select only particular metrics, instead of showcasing everything it returns by default?

Reply:
Because of line-overflow